<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gradeManage</class>
 <widget class="QWidget" name="gradeManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="gradeTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>课程号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>课程名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>得分</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>任课教师</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>251</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="searchBycId">
      <property name="text">
       <string>根据课程号查找</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addGrade_button">
      <property name="text">
       <string>添加成绩信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>230</y>
     <width>147</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="delete_button">
      <property name="text">
       <string>删除选中成绩信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="update_gradeButton">
      <property name="text">
       <string>修改选中成绩信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="student_statistics">
      <property name="text">
       <string>学生个人成绩统计</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="course_statistics">
      <property name="text">
       <string>课程成绩统计</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="export_button">
      <property name="text">
       <string>导出为Excel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>学号：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sId_lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>课程号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cId_lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>271</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="searchBysNo">
      <property name="text">
       <string>根据学号查找</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="listAll_button">
      <property name="text">
       <string>查看所有成绩</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
